{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/docs/project-overview.md

```diff
@@ -24,48 +24,75 @@ This project contains the following components:
 contained in the following files:
 
 ```
-{% if cookiecutter.include_feature_store == "yes" -%}
-├── features              <- Feature computation code (Python modules) that implements the feature transforms.
-│                         The output of these transforms get persisted as Feature Store tables. Most development
-│                         work happens here.
-│
-├── notebooks          <- Databricks notebooks that run the ML pipelines, i.e. run the features logic as part of an ETL
-│                         job that publishes to a Feature Store table. Used to drive code execution on Databricks for CI/CD.
-│                         In most cases, you do not need to modify these notebooks.
-│
-├── requirements.txt   <- Specifies Python dependencies for ML code (model training, batch inference, etc).
-│
-├── tests              <- Unit tests for the modules under `features`.
+{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}         <- Root directory. Both mono repo and poly repo are supported.
 │
+├── {{cookiecutter.project_name}}       <- Contains python code, notebooks and ML resources related to one ML project. 
+│   │
+{% if cookiecutter.include_feature_store == "yes" -%}
+│   ├── training                <- Training folder contains Notebook that trains and regisetered the model with feature store support.
+│   │
+│   ├── feature_engineering     <- Feature computation code (Python modules) that implements the feature transforms.
+│   │                              The output of these transforms get persisted as Feature Store tables. Most development
+│   │                              work happens here.
+│   │
+│   ├── validation              <- Optional model validation step before deploying a model.
+│   │
+│   ├── monitoring              <- Model monitoring, feature monitoring, etc.
+│   │
+│   ├── deployment              <- Model deployment and endpoint deployment.
+│   │   │
+│   │   ├── batch_inference     <- Batch inference code that will run as part of schduled workflow.
```

Comment:
Please fix typos.

Reply:
Please do all these same changes below, seems like it's duplicated (does it have to be?)

Reply:
The `test` folder below it has different text. That's why I have to duplicate `deployment` in both branches.